src/foraging_gui/Dialogs.py

```diff
@@ -1370,28 +1363,15 @@ def _StartCamera(self):
             self.MainWindow.Channel.CameraControl(int(1))
             time.sleep(5)
             self.camera_start_time = str(datetime.now())
-            self.MainWindow.WarningLabelCamera.setText('Camera is on!')
-            self.MainWindow.WarningLabelCamera.setStyleSheet(self.MainWindow.default_warning_color)
-            self.WarningLabelCameraOn.setText('Camera is on!')
-            self.WarningLabelCameraOn.setStyleSheet(self.MainWindow.default_warning_color)
-            self.WarningLabelLogging.setText('')
-            self.WarningLabelLogging.setStyleSheet("color: None;")
-            self.WarningLabelOpenSave.setText('')
+            logging.info('Camera is on!', extra={'tags': [self.MainWindow.warning_log_tag]})
         else:
             self.StartRecording.setStyleSheet("background-color : none")
-            self.WarningLabelCameraOn.setText('Camera is turning off')
-            self.WarningLabelCameraOn.setStyleSheet(self.MainWindow.default_warning_color)
+            logging.info('Camera is turning off', extra={'tags': self.MainWindow.warning_log_tag})
             QApplication.processEvents()
             self.MainWindow.Channel.CameraControl(int(2))
             self.camera_stop_time = str(datetime.now())
             time.sleep(5)
-            self.MainWindow.WarningLabelCamera.setText('Camera is off!')
```

Comment:
These camera warnings would appear and then disappear. Now it will be a queue, would it be valuable to include a timestamp to make it clear which happened last? I would ask the RAs

Reply:
Just added in timestamps and will ask ra's